### ISSUE TITLE
Multiline colors: implement D3's categorical colors

### ISSUE THREAD

**Issue by Jintzo** [mozilla/metrics-graphics#632]
D3 offers categorical colors for ordinal scales. I think it would be very useful if one could use them to color the lines in a multiline graph.

1. 20 instead of 5 maximum styled lines by default (see #386)
2. custom mouseover: use `d.line_id` to get the color of the current line (ability to style parts of the tooltip with the correct color)

**Comment by hamilton** [comment]
Yes, agreed - this would be a much smarter way to do this, and we're already using D3's categorical colors elsewhere.

**Comment by shikhar-scs** [comment]
@hamilton If this issue is still up, can I work on it ?
